### PR TITLE
Pending rebase: Allow limit lines to have a image at the start or end of the line

### PR DESCRIPTION
**Pull request by steprescott**
Limit line can now have an image set onto it that is then drawn at the start or end of the line.

### CODE REVIEW

**Comment by liuxuan30** [comment]
I guess you have to first clean the code style and squash some commits, and more important resolve the conflicts :)

File: Charts/Classes/Renderers/ChartYAxisRenderer.swift [hunk 1]
```diff
@@ -402,6 +406,18 @@ public class ChartYAxisRenderer: ChartAxisRendererBase
                         attributes: [NSFontAttributeName: l.valueFont, NSForegroundColorAttributeName: l.valueTextColor])
                 }
             }
+            
+            if let image = l.image {
```
Comment (liuxuan30) [inline Charts/Classes/Renderers/ChartYAxisRenderer.swift]: another line for {, typo? :)

File: Charts/Classes/Renderers/ChartYAxisRenderer.swift [hunk 2]
```diff
@@ -70,6 +70,10 @@ public class ChartYAxisRenderer: ChartAxisRendererBase
         
         let rawInterval = range / Double(labelCount)
         var interval = ChartUtils.roundToNextSignificant(number: Double(rawInterval))
+        if(_yAxis.valueFormatter?.maximumFractionDigits == 0)
```
Comment (liuxuan30) [inline Charts/Classes/Renderers/ChartYAxisRenderer.swift]: This is not about limit lines, right? It's your solution for the formatter issue. Should be another PR if think it's good to everyone

Comment (danielgindi) [inline Charts/Classes/Renderers/ChartYAxisRenderer.swift]: Yes please separate subjects to different PRs. Every subject should be on a different branch - and then creating a PR from that branch will create a separate PR.

**Comment by danielgindi** [comment]
Thank you for this effort!
However there are a few things keeping us from merging it:
1. There's code in there which is not related to the subject of the PR
2. Commits should be squashed
3. Most importantly: Resolve merge conflicts. If you rebase everything on the latest master it should be fine. This could be done by adding the original repo `danielgindi/ios-charts` as a `remote` in your fork, and pulling from it to your repo.

**Comment by danielgindi** [comment]
Closing due to no response....
